### ISSUE TITLE
Case issues cause compilation to fail in ArchLinux

### ISSUE THREAD

**Issue by zxp19821005** [upscayl/upscayl#1004]
### Checklist

- [X] I have **tried ALL** of the steps mentioned in the [troubleshooting guide](https://github.com/upscayl/upscayl/wiki/Troubleshooting).
- [X] I have **searched** for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Describe the Bug

I've ported it to ArchLinux AUR: [upscayl-git](https://aur.archlinux.org/packages/upscayl-git)
But someone reported that the compilation failed：https://pastebin.com/LEsBsA7G
I checked it and found it was a case error：
https://github.com/upscayl/upscayl/blob/main/renderer/components/sidebar/index.tsx

https://github.com/upscayl/upscayl/blob/ddd286574b85980ebfea110b275d8f6f339fce69/renderer/components/sidebar/index.tsx#L31
https://github.com/upscayl/upscayl/blob/ddd286574b85980ebfea110b275d8f6f339fce69/renderer/components/sidebar/index.tsx#L33
https://github.com/upscayl/upscayl/blob/ddd286574b85980ebfea110b275d8f6f339fce69/renderer/components/sidebar/index.tsx#L34
https://github.com/upscayl/upscayl/tree/main/renderer/components
![image](https://github.com/user-attachments/assets/339e4724-c35b-4464-a2c1-3f728472e54d)
![image](https://github.com/user-attachments/assets/c769244b-242b-49e8-b1f4-d4568eda2609)
I've fixed it in my newest commit on AUR.

### To Reproduce

Just build it.

### Upscayl Version (or commit hash)

ddd2865

### Platform

Linux

### OS Version

6.11.3-arch1-1

### GPU Name

Iris Xe Graphics

### Logs

```sh
https://pastebin.com/LEsBsA7G
```

**Comment by NayamAmarshe** [comment]
Please don't build on the main branch. It's not always stable.

**Comment by zxp19821005** [comment]
@NayamAmarshe That doesn't matter.In Archlinux, `-git` means latest but unstable.I've just feedback.

**Comment by aaronliu0130** [comment]
@zxp19821005 We recently merged #1005. Can you check if that fixed it?

**Comment by zxp19821005** [comment]
@aaronliu0130
Still got errors:
```
==> 正在开始 build()...
v18.20.4 is already installed.
Now using node v18.20.4 (npm v10.7.0)
Now using node v18.20.4 (npm v10.7.0)
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated @humanwhocodes/config-array@0.11.14: Use @eslint/config-array instead

added 899 packages in 19s

251 packages are looking for funding
  run `npm fund` for details
npm notice
npm notice New minor version of npm available! 10.7.0 -> 10.9.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.9.0
npm notice To update run: npm install -g npm@10.9.0
npm notice

added 9 packages in 3s

254 packages are looking for funding
  run `npm fund` for details

> upscayl@2.11.5 tsc
> tsc


> upscayl@2.11.5 build
> tsc && next build renderer

Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

  ▲ Next.js 14.2.10
  - Experiments (use with caution):
    · externalDir

   Linting and checking validity of types  .Failed to compile.

./components/sidebar/index.tsx:31:20
Type error: Cannot find module '../Footer' or its corresponding type declarations.

  29 | import UpscaylSteps from "./upscayl-tab/upscayl-steps";
  30 | import SettingsTab from "./settings-tab";
> 31 | import Footer from "../Footer";
     |                    ^
  32 | import { NewsModal } from "../news-modal";
  33 | import Tabs from "../Tabs";
  34 | import Header from "../Header";
==> 错误： 在 build() 中发生一个错误。
    正在放弃...
```

**Comment by aaronliu0130** [comment]
@zxp19821005 That's not what our newest code ( https://github.com/upscayl/upscayl/blob/0b78821aeb14ed557695de85c40bc75327cf1816/renderer/components/sidebar/index.tsx ) looks like. Why do you have a nested components folder under the components folder? Are you sure you're trying to build through the makepkg command/have updated your git repository?
